lib/proto/parse-element.js

```diff
@@ -2,14 +2,14 @@ module.exports = function(el) {
   switch (el.tagName.toLowerCase()) {
     case "a":
       // only attach link if el does not already have link attached
-      if (!el.hasAttribute("data-pjax-click-state")) {
+      if (!el.hasAttribute("data-pjax-state")) {
```

Comment:
This, and line 12 could be abstracted to use `var attrState = "data-pjax-state"`, as in `attach-link`

Reply:
True.